atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -65,9 +65,9 @@ def __init__(self, tasker, workflow, registries=None, group=True, goarch=None):
         self.group = get_group_manifests(self.workflow, group)
 
         plat_des_fallback = []
-        for platform in goarch:
+        for platform, architecture in (goarch or {}).items():
             plat_dic = {'platform': platform,
-                        'architecture': goarch[platform]}
+                        'architecture': architecture}
```

Comment:
what's the reason for this change? if you want to handle the case where goarch=None, couldn't this change just as easily be 'for platform in goarch or {}:'
I don't really object, I'm just curious about why one form is preferred over the other.

Reply:
the same change was done in orchestrator, as Tim wanted to use items() since we are using both key&value